test/jdk/javax/swing/JSpinner/4515999/JSpinnerMouseAndKeyPressTest.java

```diff
@@ -0,0 +1,169 @@
+import java.awt.ComponentOrientation;
+import java.awt.Point;
+import java.awt.Robot;
+import java.awt.event.InputEvent;
+import java.awt.event.KeyEvent;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.JSpinner;
+import javax.swing.SpinnerDateModel;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+
+
+import static javax.swing.UIManager.getInstalledLookAndFeels;
+
+/*
+ * @test
+ * @key headful
+ * @bug 4515999
+ * @summary Check whether incrementing dates via the keyboard (up/down) gives
+ * the same results as using mouse press on the arrow buttons in a JSpinner.
+ * @run main JSpinnerMouseAndKeyPressTest
+ */
+public class JSpinnerMouseAndKeyPressTest {
+    //2 days in milliseconds
```

Comment:
Please add a space after `//` according to [the guidelines](http://cr.openjdk.java.net/~alundblad/styleguide/index-v6.html#toc-whitespace). In other places too.